app.chunker.chunker.yml

```diff
@@ -0,0 +1,42 @@
+id: app.chunker.chunker
+runtime: org.freedesktop.Platform
+runtime-version: "25.08"
+base: "org.electronjs.Electron2.BaseApp"
+base-version: "25.08"
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - "org.freedesktop.Sdk.Extension.openjdk17"
+command: app.chunker.chunker
+finish-args:
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --share=ipc
+  - --device=dri
+  - --env=ELECTRON_TRASH=gio
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - "/usr/lib/sdk/openjdk17/install.sh"
+  - name: chunker
+    buildsystem: simple
+    build-commands:
+      - "mkdir -p /app/chunker"
+      - "cp -r * /app/chunker"
+      - patch-desktop-filename /app/chunker/resources/app.asar
+      - "chmod +x /app/chunker/chunker-electron"
+      - "chmod +x /app/chunker/resources/chunker-cli/bin/chunker-cli"
+      - "install -Dm755 chunker.sh /app/bin/app.chunker.chunker"
+      - "install -Dm644 resources/flathub/app.chunker.chunker.metainfo.xml /app/share/metainfo/app.chunker.chunker.metainfo.xml"
+      - "install -Dm644 resources/flathub/app.chunker.chunker.desktop /app/share/applications/app.chunker.chunker.desktop"
+      - "install -Dm644 resources/flathub/logo.png /app/share/icons/hicolor/512x512/apps/app.chunker.chunker.png"
+    sources:
+      - type: archive
+        only-arches:
+          - "x86_64"
+        url: https://github.com/HiveGamesOSS/Chunker/releases/download/1.13.0/Chunker-1.13.0-amd64-linux.zip
```

Comment:
This should be built from source.

https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source